<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 40, 40);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>96</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(234, 234, 234);</string>
   </property>
   <property name="text">
    <string>Abrir Archivo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>541</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Imagenes.qrc">:/new/prefix1/higgsch.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>73</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(234, 234, 234);</string>
   </property>
   <item>
    <property name="text">
     <string>Editar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Efecto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mejora</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="widgetMejora" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>431</width>
     <height>150</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>371</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>Brillo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="minimum">
        <number>-25</number>
       </property>
       <property name="maximum">
        <number>25</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>Saturacion</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetEfectos" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButtonFiltro3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFiltro1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFiltro2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFiltro4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="sliderComparar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/Flechas2.png);</string>
   </property>
   <property name="maximum">
    <number>541</number>
   </property>
   <property name="value">
    <number>270</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>87</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(234, 234, 234);</string>
   </property>
   <property name="text">
    <string>Comparar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>271</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(234, 234, 234);</string>
   </property>
   <property name="text">
    <string>Antes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>69</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Corbel</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(234, 234, 234);</string>
   </property>
   <property name="text">
    <string>Despues</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetEditar" native="true">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>281</width>
     <height>501</height>
    </rect>
   </property>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234, 234, 234);</string>
    </property>
    <item>
     <property name="text">
      <string>AND</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XOR</string>
     </property>
    </item>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234, 234, 234);</string>
    </property>
    <property name="text">
     <string>Operadores</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234, 234, 234);</string>
    </property>
    <property name="text">
     <string>Corrimiento</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>251</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QLCDNumber" name="lcdRCorrimiento">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="sliderRCorrimientro">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QLCDNumber" name="lcdGCorrimiento">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSlider" name="sliderGCorrimientro">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" rowspan="2">
      <widget class="QLCDNumber" name="lcdBCorrimiento">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSlider" name="sliderBCorrimientro">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>45</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234, 234, 234);</string>
    </property>
    <item>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>73</y>
      <width>251</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QLCDNumber" name="lcdROperador">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Hex</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="sliderROperador">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QLCDNumber" name="lcdGOperador">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Hex</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSlider" name="sliderGOperador">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" rowspan="2">
      <widget class="QLCDNumber" name="lcdBOperador">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Hex</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSlider" name="sliderBOperador">
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 234, 234);</string>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
